Class-8-(Intro-to-Spring-and-Controllers).md

```diff
@@ -0,0 +1,34 @@
+# Class 8: Introduction to Spring Boot, MVC, and Controllers
+
+## In the prep work this week, the students learned
+
+1. About the MVC design pattern.
+1. What the Spring framework is.
+1. How to initialize a Spring Boot project.
+1. How to handle routing and write controllers in Spring
+
+## Announcements
+
+1. Check with your course manager for any important announcements.
+
+## Large Group Time (Instructor)
+
+### Class 8 Topics That Require Careful Attention
+
+1. The Java/Gradle development environment can be tricky. Some students may encounter that we were not able to anticiapte in the text. Some minor versions may be different than what is listed in the 
+text. 
+
+   1. Java 13 and Gradle 5.6 are the most crucial to have. 
```

Comment:
For the Thymeleaf content, we should have the students get used to using Gradle 6.x.

Also, maybe this is a good place to have the instructors go over the bug in the ``gradle-wrapper.properties`` file (having to manually update 5.6.x to 6.0).